opentelemetry/trace/main.go

```diff
@@ -63,6 +60,7 @@ func main() {
 
 		return nil
 	}(ctx)
+	exporter.Shutdown(ctx) // flush any pending spans
```

Comment:
Should this be called with `defer`?

Reply:
I had wondered, but `ctx` was defined further below.  Hmm, I'm not familiar with this code, but I wonder if we're better using the exporter options?

Reply:
I don't see [any combination of bundler options][opts] that can be set to disable the bundling.

[opts]: github.com/GoogleCloudPlatform/opentelemetry-operations-go/exporter/trace/trace.go

Reply:
Defer looks good to me. Thank you for checking!